### ISSUE TITLE
1.2.5 requirements for Audio description when soundtrack of video has no gaps

### ISSUE THREAD

**Issue by detlevhfischer** [w3c/wcag#909]
I would like to get the view of the WG whether audio description (AD) is required for instructional videos with voice over that have information also in the visual content (e.g. you see an animation where a form is filled in), but there is continuous audio and thus not sufficient gaps for inserting AD (or the gaps are too small for a useful AD). So far, we have not failed such videos on the basis that a workable solution would actually require an extended AD, which is only required at level AAA (SC 1.2.7).

One indication is given the (non-normative) G78 technique:

> "Since it is not helpful to have this new information obscure key audio information in the original sound track (or be obscured by loud sound effects), the new information is added during pauses in dialogue and sound effects. This limits the amount of supplementary information that can be added to the program."

There is no failure assigned to 1.2.5 that would clearly state that when image content is not sufficiently covered in a continuous audio track but the image content would be important for understanding, the content fails.

The difficult question which also arises, is whether in cases where an AD would clearly be necessary, but the video has no space for it, then SC 1.2.3 (AD or media alternative) would need to be considered in conjunctiom - along the lines of  "OK, if there is no space on the audio track, then at least provide a separate text alternative". Would both 1.2.3 and 1.2.5 then pass?

**Comment by bruce-usab** [comment]
I agree that this is an example where minimal conformance can result with an outcome that is not very accessible.  We have the potential for that with several SC, so I am not too troubled by the gap.  Maybe Silver will help?

See second note in the [definition for audio description](http://www.w3.org/TR/WCAG21/#dfn-audio-descriptions):  “Narration is added during existing pauses in dialogue.”

From that definition, not failing such videos because there is continuous audio and thus not sufficient gaps for inserting AD (or the gaps are too small for a useful AD) is correct.

**Comment by alastc** [comment]
This is an old discussion thread, so closing.

(And I agree with Bruce's answer. We could also consider including a technique for including the information that would be in AD in a voiceover.)

**Comment by mitchellevan** [comment]
This issue was closed appropriately, because it was redundant to discussions in other issues.

Those other issues are currently still open, so I'm linking to them...

* The main discussion: https://github.com/w3c/wcag/issues/1768
* Part of another discussion: https://github.com/w3c/wcag/issues/782#issuecomment-501285200
